sda-commons-web-autoconfigure/src/test/java/org/sdase/commons/spring/boot/web/logging/HttpRequestLoggingJsonTest.java

```diff
@@ -100,6 +103,17 @@ private ResponseEntity<String> executeRequest(String path, int port) {
     return client.getForEntity(String.format("http://localhost:%d%s", port, path), String.class);
   }
 
+  private ResponseEntity<String> executeRequestWithHeader(String path, int port) {
+    var headers = new HttpHeaders();
+    headers.set("trace-token", "shrek");
```

Comment:
Are we case sensitive?

Reply:
Ah ups, no we are not. Also I wanted to replace the text with more appropriate test data, overlooked that :/